deploy/README.md

```diff
@@ -214,7 +244,7 @@ default, the plan deploys VMs with a single NIC, so it should be easy to spot.
 Alternatively, one can run
 
 ```
-    # ifconfig eth0 | grep 'inet ' | awk '{print $2}'
+    # ip -o -4 addr list eth0 | awk '{print $4}' | cut -d/ -f1
```

Comment:
I don't have this problem - the image specified in plan/cluster.yml is fedora34, which seems to include both `ifconfig` and `ip`. Still, `ifconfig` has been deprecated for years, so good move.

Reply:
Throwing `hostname -I` in the ring :)